cloud/services/resourceskus/sku.go

```diff
@@ -52,6 +52,14 @@ const (
 	EphemeralOSDisk = "EphemeralOSDiskSupported"
 	// AcceleratedNetworking identifies the capability for accelerated networking support.
 	AcceleratedNetworking = "AcceleratedNetworkingEnabled"
+	//VCPUs identifies the capability for the number of vCPUS.
+	VCPUs = "vCPUs"
+	// MemoryGB identifies the capability for memory Size.
+	MemoryGB = "MemoryGB"
+	// MinimumVCPUS is the minimum vCPUS allowed.
+	MinimumVCPUS = 2
+	// MinimumMemory is the minimum memory allowed.
+	MinimumMemory = 4
```

Comment:
should be 2 to be consistent with https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin

Reply:
😅